app/models/organization.rb

```diff
@@ -65,8 +65,7 @@ class Organization < ApplicationRecord
         where("super_users.identity_id = ? or service_providers.identity_id = ?", identity_id, identity_id).
         references(:super_users, :service_providers).
         distinct(:organizations).ids
-      ),
```

Comment:
Should there also be a distinction made in the filter's dropdown that denotes inactive organizations?